src/components/techcard/TechCard.module.scss

```diff
@@ -1,5 +1,5 @@
 .card {
-   flex-basis: 30%;
+   flex-basis: 50%;
```

Comment:
This is done to ensure that note more than 2 cards are displayed on the same row irrespective of the screen size